stl/inc/ranges

```diff
@@ -805,8 +700,12 @@ namespace ranges {
         };
     };
 
+    template <class _Derived>
+        requires is_class_v<_Derived> && same_as<_Derived, remove_cv_t<_Derived>>
+    class range_adaptor_closure;

```

Comment:
I've noticed that none of your change in `<ranges>` is guarded by `#if _HAS_CXX23`. Not sure whether this is OK.

Reply:
Thank you very much for that hint! I fixed that.